pkg/cmd/config.go

```diff
@@ -812,13 +812,15 @@ addons:
 #     bastion: '4.3.2.1'
 #     bastionPort: 22  # can be left out if using the default (22)
 #     bastionUser: 'root'  # can be left out if using the default ('root')
+#     bastionHostPublicKey: "AAAAC3NzaC1lZDI1NTE5AAAAIGpmWkI5dl7GB3E1hB9LDuju87x9hX5Umw9fih+xXNU+"
```

Comment:
Can you add a comment describing this field similar to what we have for other fields?